conans/client/generators/cmake_find_package.py

```diff
@@ -19,14 +19,15 @@ class CMakeFindPackageGenerator(Generator):
 
         include(FindPackageHandleStandardArgs)
 
-        conan_message(STATUS "Conan: Using autogenerated Find{name}.cmake")
+        conan_message(STATUS "Conan: Using autogenerated Find{filename}.cmake")
         # Global approach
-        set({name}_FOUND 1)
+        set({filename}_FOUND 1)
+        {set_found_filename_uppercase}
```

Comment:
please implement this as
```suggestion
        set({filename_upper}_FOUND 1)
```